<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>761</width>
      <height>421</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Match Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stadium</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Team 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Team 2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Team 1 Score</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Team 2 Score</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>70</y>
      <width>202</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="showAllButton">
       <property name="text">
        <string>Show all matches</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addNewButton">
       <property name="text">
        <string>Add new match</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete match</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="winsButton">
       <property name="text">
        <string>Teams with n or more wins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="drawsButton">
       <property name="text">
        <string>Display draws</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="graphButton">
       <property name="text">
        <string>Show stadiums graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Select team</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMatches">
    <property name="title">
     <string>Matches</string>
    </property>
   </widget>
   <addaction name="menuMatches"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
